src/main/java/org/terasology/blockdetector/systems/BlockDetectorSystemImpl.java

```diff
@@ -104,33 +103,26 @@
     private Set<Vector3i> detectedBlocks = new HashSet<>();
 
     /**
-    * This set the local player for the tests
+    * This sets the local player for the tests
     * The dummy local player will make the system work and does not throw a null exception
-    * @param set the local player that going to be set through the test method
     */
     public void setLocalPlayer(LocalPlayer set) {
         this.localPlayer = set;
     }
 
-    /**
-     * This provides the value for the test
-     * @return a float of timeSinceLastUpdate
-     */
     public float getTimeSinceLastUpdate() {
         return this.timeSinceLastUpdate;
     }
 
     /**
-    * This set the timeSinceLastUpdate value for the tests
-    * @param value a float of the value that going to be assigned to the variable
+    * This sets the timeSinceLastUpdate value for the tests
     */
     public void setTimeSinceLastUpdate(float value) {
         this.timeSinceLastUpdate = value;
     }
 
     /**
     * This provides the DetectorData for the tests
-    * @return a Map of DetectorData
```

Comment:
Regarding this change:
When writing javadocs for accessors (get/set funtions) there are 3 broad ways:
- Don't write javadocs, since the function name is usually self-eplanatory. This is not recommended if you're documenting an API, however.
- Only write the descripition, and leave out the return value (what you've done)
- Write the description, and a redundant return value (what was done originally). This is the most complete form, and is the only case where redunddant javadocs are acceptable.
 
That being said, the return line is actually useful here, because it tells us the type of value returned - `A map`. So, I would request you revert this one change.